_pages/interviews.md

```diff
@@ -42,7 +42,7 @@ Now, **this isn’t an excuse to be a jerk!** Being selective is important, but
 <a name="kick-off"></a>
 ## Kicking off the interview
```

Comment:
I *think* we shouldn't need this hrefless link. The headings *should* be converted to have IDs, so I *think* we should be able to link directly to something like #kicking-off-the-interview

Reply:
I think we shouldn't need it either, but there lies what seems like a deep rabbit hole.  I'm still dirty from the Docker and CircleCI rabbit holes, so I will probably let these be.